src/config.rs

```diff
@@ -18,21 +20,37 @@ pub struct E4EDMConfig {
 
 impl E4EDMConfig {
     pub fn save(&self) -> Result<()> {
-        let config_str = toml::to_string(&self)?;
+        let config_str = serde_json::to_string(&self)?;
```

Comment:
May want to use `to_string_pretty` instead, I currently find the JSON file very difficult to read.